curvefs/src/mds/topology/topology.cpp

```diff
@@ -938,6 +938,17 @@ TopoStatusCode TopologyImpl::ChooseSingleMetaServerInZone(ZoneIdType zoneId,
     return TopoStatusCode::TOPO_OK;
 }
 
+uint32_t TopologyImpl::GetPartitionNumberOfFs(FsIdType fsId) {
+    ReadLockGuard rlockPartition(partitionMutex_);
+    uint32_t pNumber = 0;
+    for (auto it : partitionMap_) {
```

Comment:
`for (const auto& it ...` to avoid unnecessary copy

Reply:
> `for (const auto& it ...` to avoid unnecessary copy

fixed